src/pages/SelectLocationsPage.js

```diff
@@ -136,7 +136,7 @@ class SelectlocationsPage extends React.Component {
     axios
       .get(`/locations/${selectedProvince}`)
       .then(locs => {
-        if (locs && Array.isArray(locs)) {
+        if (locs && Array.isArray(locs.data)) {
```

Comment:
this line was changed by latest PR by Adam

Reply:
Yes, i need to merge with the changes that Adam made.

Reply:
Just rebase on master, My changes are in there now :egg: